using_listwheelscrollview/lib/listwheel.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'roundcontainer.dart';
+
+class Listwheel extends StatefulWidget {
+  @override
+  _ListwheelState createState() => _ListwheelState();
+}
+
+class _ListwheelState extends State<Listwheel> {
+  double size = 28;
+  int radius = 8;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.white70,
+      body: Container(
+        child: ListWheelScrollView(
+          children: [
+            NewWidget(

```

Comment:
Extra the list into a function that returns this array. So this build function is simpler.

Reply:
That function should iterate over a for loop and use another function to pass in the icon and text as argument. Since this part is common.

```
class _ListwheelState extends State<Listwheel> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      backgroundColor: Colors.white70,
      body: Container(
        child: ListWheelScrollView(
          children: getChildren(),
          squeeze: 1.0,
          itemExtent: 180,
          diameterRatio: 1.9,
          offAxisFraction: -0.5,
        ),
      ),
    );
  }
}

List<NewWidget> getChildren() {
  double size = 28;
  final titles = ['Brush', 'Style'];
  final icons = [Icons.brush, Icons.style];

  final children = List<NewWidget>()

  for (var i = 0; i < titles.length; i++) {
    children.add(getNewWidget(size, icons[i], titles[i]));
  }
  return children;
}

NewWidget getNewWidget(size, icon, title) {
  return NewWidget(
      size: size,
      l: Icon(
        icon,
        color: Colors.white54,
        size: size,
      ),
      s: title);
}

```

Reply:
ok sir.